index.js

```diff
@@ -177,7 +194,7 @@ class Replicate {
     const response = await endpoint();
     yield response.results;
     if (response.next) {
-      const nextPage = () => this.request(response.next, { method: 'GET' });
+      const nextPage = () => this.request(response.next, { method: 'GET' }).then((r) => r.json());
```

Comment:
Tiny style preference: if we use `async..await` instead of `Promise.then` semantics, it's a lot easier to spot where the asynchronous parts of something are. Not a blocker, though.